ada/language/ast.py

```diff
@@ -6412,6 +6412,32 @@ class DefiningName(Name):
         doc="Returns this DefiningName's basic declaration"
     )
 
+    @langkit_property(public=False, return_type=AdaNode.entity.array)
+    def find_all_refs_in(x=AdaNode.entity):
+        """
+        Searches all references to this defining name in the given node.
```

Comment:
So if I understand correctly, its looking for all refs in the given `x` subtree, right? (not just `x` as a node)